### ISSUE TITLE
6.2.6 installation from PyPI is broken

### ISSUE THREAD

**Issue by Tenzer** [thumbor-community/aws#117]
This is what I see when I try to install version 6.2.6 (the latest available version) from PyPI:
```
(tmp-707a197e477eaec) [jeppe@reventon ~]$ pip install tc_aws
Collecting tc_aws
  Downloading https://files.pythonhosted.org/packages/b8/34/c32432665513740659bf7af83d2f9ee888f65c53258e095149c46d3b2de2/tc_aws-6.2.6.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/12/s65kxmp934lfst0r8m015psm0000gn/T/pip-install-zzaflaj5/tc-aws/setup.py", line 31, in <module>
        long_description=readme(),
      File "/private/var/folders/12/s65kxmp934lfst0r8m015psm0000gn/T/pip-install-zzaflaj5/tc-aws/setup.py", line 24, in readme
        with open('Readme.md') as f:
    FileNotFoundError: [Errno 2] No such file or directory: 'Readme.md'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/12/s65kxmp934lfst0r8m015psm0000gn/T/pip-install-zzaflaj5/tc-aws/
```

This happens because `Readme.md` is not included in the `.tar.gz` file:
```
(tmp-707a197e477eaec) [jeppe@reventon ~]$ curl -s https://files.pythonhosted.org/packages/b8/34/c32432665513740659bf7af83d2f9ee888f65c53258e095149c46d3b2de2/tc_aws-6.2.6.tar.gz | tar t
tc_aws-6.2.6/
tc_aws-6.2.6/setup.py
tc_aws-6.2.6/PKG-INFO
tc_aws-6.2.6/tc_aws.egg-info/
tc_aws-6.2.6/tc_aws.egg-info/dependency_links.txt
tc_aws-6.2.6/tc_aws.egg-info/SOURCES.txt
tc_aws-6.2.6/tc_aws.egg-info/top_level.txt
tc_aws-6.2.6/tc_aws.egg-info/PKG-INFO
tc_aws-6.2.6/tc_aws.egg-info/requires.txt
tc_aws-6.2.6/tc_aws.egg-info/not-zip-safe
tc_aws-6.2.6/setup.cfg
tc_aws-6.2.6/tc_aws/
tc_aws-6.2.6/tc_aws/result_storages/
tc_aws-6.2.6/tc_aws/result_storages/s3_storage.py
tc_aws-6.2.6/tc_aws/result_storages/__init__.py
tc_aws-6.2.6/tc_aws/aws/
tc_aws-6.2.6/tc_aws/aws/storage.py
tc_aws-6.2.6/tc_aws/aws/__init__.py
tc_aws-6.2.6/tc_aws/aws/bucket.py
tc_aws-6.2.6/tc_aws/__init__.py
tc_aws-6.2.6/tc_aws/storages/
tc_aws-6.2.6/tc_aws/storages/s3_storage.py
tc_aws-6.2.6/tc_aws/storages/__init__.py
tc_aws-6.2.6/tc_aws/loaders/
tc_aws-6.2.6/tc_aws/loaders/presigning_loader.py
tc_aws-6.2.6/tc_aws/loaders/__init__.py
tc_aws-6.2.6/tc_aws/loaders/s3_loader.py
tc_aws-6.2.6/vows/
tc_aws-6.2.6/vows/loader_vows.py
tc_aws-6.2.6/vows/storage_vows.py
tc_aws-6.2.6/vows/fixtures/
tc_aws-6.2.6/vows/fixtures/storage_fixture.py
tc_aws-6.2.6/vows/fixtures/__init__.py
tc_aws-6.2.6/vows/__init__.py
tc_aws-6.2.6/vows/s3_loader_vows.py
tc_aws-6.2.6/vows/presigning_loader_vows.py
tc_aws-6.2.6/vows/result_storage_vows.py
```

It looks like commit 437d3137e1ab007914f4ab0a0d323fbc408982c8 has broken this.

You probably want to add a `MANIFEST.in` file with `include Readme.md` to the root of the repository, as suggested here: http://python-packaging.readthedocs.io/en/latest/metadata.html.

**Comment by Bladrak** [comment]
Thanks for reporting, I'll take a look!

**Comment by Bladrak** [comment]
Fixed in 6.2.7, please re-open the issue if it's not the case (but should be fine, I just tested it :) )
